package.json

```diff
@@ -9,6 +9,9 @@
     "css",
     "libsass"
   ],
+  "engines": {
+    "node": ">=10.24.1"
```

Comment:
Is `fs.promises` in node 10?

Reply:
Yes

Reply:
https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fs_promises_api